src/index.ts

```diff
@@ -1,65 +1,127 @@
 import { DurableObject } from 'cloudflare:workers';
 
-/**
- * Welcome to Cloudflare Workers! This is your first Durable Objects application.
- *
- * - Run `npm run dev` in your terminal to start a development server
- * - Open a browser tab at http://localhost:8787/ to see your Durable Object in action
- * - Run `npm run deploy` to publish your application
- *
- * Bind resources to your worker in `wrangler.toml`. After adding bindings, a type definition for the
- * `Env` object can be regenerated with `npm run cf-typegen`.
- *
- * Learn more at https://developers.cloudflare.com/durable-objects
- */
-
-/** A Durable Object's behavior is defined in an exported Javascript class */
+// Worker
+export default {
+	async fetch(request: Request, env: Env, ctx: ExecutionContext): Promise<Response> {
+		if (request.url.endsWith('/websocket')) {
+			// Expect to receive a WebSocket Upgrade request.
+			// If there is one, accept the request and return a WebSocket Response.
+			const upgradeHeader = request.headers.get('Upgrade');
+			if (!upgradeHeader || upgradeHeader !== 'websocket') {
+				return new Response('Durable Object expected Upgrade: websocket', {
+					status: 426,
+				});
+			}
+
+			if (request.method !== 'GET') {
+				return new Response('Durable Object expected GET method', {
+					status: 400,
+				});
+			}
+
+			// This example will refer to the same Durable Object,
+			// since the name "foo" is hardcoded.
+			let id = env.WEBSOCKET_HIBERNATION_SERVER.idFromName('foo');
+			let stub = env.WEBSOCKET_HIBERNATION_SERVER.get(id);
+
+			return stub.fetch(request);
+		}
+
+		return new Response(null, {
```

Comment:
If the intention is to have people clone and run this example, I'd suggest having the default path return usage :slightly_smiling_face:. 

For example:

```
Supported endpoints:

/websocket
    - Expects a WebSocket upgrade request
``` 
I've previously [deployed a DO](https://typescript.milanmiladinovic.workers.dev/) that does this.

This way, when sending an initial fetch, they won't be greeted with a `Bad Request` (which might make them think the deployment isn't working).